README.adoc

```diff
@@ -456,4 +456,32 @@ To unify upper/lowercase within the (EN) version, we use the _Chicago manual of
 For the German (DE) version, we don't use punctuation at the end of bullet-list items, unless on ends-of-sentences.
 
 
+[[handling-references]]
+=== Based on Established Content And References
+We strive to base all content on established content by providing references to original sources.
 
+In practice, this means that we reference the original source of a learning goal, if it is based on a book, a paper, a standard or similar.
+We use a distinct section for every learning goal:
+
+ 
+```
+===== {references}
+<<starkelorz>>
+``` 
+
+Example (abbreviated)
+----
+/ tag::EN[]
```

Comment:
We need one more `/` here. 😀

Reply:
removed the tag::EN completely, not useful here